extension/utils.go

```diff
@@ -73,6 +73,15 @@ func GetS3IPFSExtensionBucketHost() string {
 	return s3BucketHost
 }
 
+// GetDynamoRegion returns the dynamoDB region
```

Comment:
not needed, check the docs link, it passes AWS_REGION automatically if set

Reply:
still need it to build the docker image, we can change the name to AWS_REGION, but we have to assign the default value of region string and pass it to session in order to build the image

Reply:
what happens if you take it out? there should be no such dependence on hardcoded values during build and ideally also during runtime

Reply:
I keep getting error during docker build, maybe @jumde knows better?
```
controller/controller.go:58:5: undefined: err
controller/controller.go:59:55: undefined: err
controller/controller.go:60:27: undefined: err
make: *** [Makefile:6: build] Error 2
```

Reply:
how do we build or call the Dockerfile?

Reply:
i'm currently building on my local and push to ECR, will create a jenkins pipeline to do it after migration is done

Reply:
I mean what commands do you use

Reply:
just build normally inside this folder `docker build -t go-updater .`

Reply:
let's add a `build-docker` target to the Makefile